src/views/DocumentList.vue

```diff
@@ -1,113 +1,192 @@
 <template>
-<div class='card'>
+  <div class='card'>
     <div class='card-content'>
-        <div class='media'>
-            <div class='media-content'>
-                <div class='field is-grouped'>
-                    <div class='control' v-if='branchNames.length'>
-                        <div class='select is-small'>
-                            <select v-model='selectedBranch' title='Switch branch' class='branch-selection' @change='switchBranch()'>
-                                <option v-for='name in branchNames' :key='name' :value='name'>
-                                    {{ name }}
-                                </option>
-                            </select>
-                        </div>
-                    </div>
-                    <p class='control'>
-                        <button class='button is-small download-doc-button' :class='{"is-loading":isDownloadLoading}' title='Download document' data-qa='download-document'>
-                            <span class='icon is-small'>
-                                <i class='fas fa-download'></i>
-                            </span>
-                        </button>
-                    </p>
-                </div>
+      <div class='media'>
+        <div class='media-content'>
+          <div class='field is-grouped'>
+            <div class='control'
+                 v-if='branchNames.length'>
+              <div class='select is-small'>
+                <select v-model='selectedBranch'
+                        title='Switch branch'
+                        class='branch-selection'
+                        @change='switchBranch()'>
+                  <option v-for='name in branchNames'
+                          :key='name'
+                          :value='name'>
+                    {{ name }}
+                  </option>
+                </select>
+              </div>
             </div>
+            <p class='control'>
+              <button class='button is-small download-doc-button'
+                      :class='{"is-loading":isDownloadLoading}'
+                      @click="downloadDoc()"
+                      title='Download document'
+                      data-qa='download-document'>
+                <span class='icon is-small'>
+                    <i class='fas fa-download'></i>
+                </span>
+              </button>
+            </p>
+          </div>
         </div>
-        <hr />
-        <table class='table is-bordered is-fullwidth is-size-7 document-list-table is-hoverable vectors-table'>
-            <thead>
+      </div>
+
+      <hr/>
+
+      <div class="content document-list-wrapper"
+           v-show="!loadingDocCounter">
+        <div class="card">
+          <div class="card-content">
+            <div class="content">
+              <table class='table is-bordered is-fullwidth is-size-7 document-list-table is-hoverable vectors-table'>
```

Comment:
This pattern should be blocked by a linter, we use double quotes for HTML attribute values delimiters

Note: For the entire file

```suggestion
              <table class="table is-bordered is-fullwidth is-size-7 document-list-table is-hoverable vectors-table">
```

Reply:
Done